### PR TITLE
For #23614 - Add default values for Fenix specific parameters

### PR DESCRIPTION
**Pull request by ahal**
Currently an exception will be raised if values for these parameters are not supplied. This will be the case when running with the default set of Parameters (e.g via `taskgraph full` without an explicit parameters file).

This PR adds defaults to these parameters (which also requires updating to the latest taskgraph).

### CODE REVIEW

**Comment by ahal** [comment]
Guessing the docker image task failure may be a result of updateing `taskgraph`. Some investigation needed.

**Comment by ahal** [comment]
> Guessing the docker image task failure may be a result of updateing `taskgraph`. Some investigation needed.

Nvm, I was just based on a really old revision that didn't have the `taskcluster` Python 2 fix yet.

File: taskcluster/fenix_taskgraph/parameters.py [hunk 1]
```diff
@@ -14,13 +14,23 @@
...
+        "release_type": "",
+        "next_version": "",
+        "version": "",
+    }
+
+
 extend_parameters_schema({
     Required("pull_request_number"): Any(All(int, Range(min=1)), None),
     Required("release_type"): text_type,
     Optional("shipping_phase"): Any('build', 'ship', None),
```
Comment (jcristau) [inline taskcluster/fenix_taskgraph/parameters.py:30]: Does `shipping_phase` not need a default?

Comment (ahal) [reply taskcluster/fenix_taskgraph/parameters.py:30]: Apparently not? Maybe we use `config.params.get("shipping_phase")` everywhere. I guess it doesn't hurt to add one in anyway.

**Comment by ahal** [comment]
Looks like Fenix is about to cut a new beta branch, so going to hold off landing this for the time being.

**Comment by ahal** [comment]
Latest push updates revision to an even newer taskgraph which includes a fix for an issue we found yesterday due to unicode in commit message.

**Pull request merged**